tasklist/client/src/Tasks/CollapsiblePanel/SearchParamNavLink/styles.module.scss

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+@use '@carbon/layout';
+@use '@carbon/type';
```

Comment:
This is not used

Reply:
Because github doesn't highlight the lines. I am referring to `@use '@carbon/type'`

Reply:
It does though, when you click just in a single line it refers to the first line above the comment you can also click and drag to refer to more than 1 line like the screenshot below:
![image](https://github.com/camunda/zeebe/assets/1022916/7ced63a5-fc7b-4a90-8827-044909dffc1a)